src/main/java/org/broadinstitute/dsm/route/AssignParticipantRoute.java

```diff
@@ -211,7 +211,8 @@ public void doNotification(String email, String message, String recordId, String
     public String getDDPParticipantId(@NonNull String participantId) {
         SimpleResult results = inTransaction((conn) -> {
             SimpleResult dbVals = new SimpleResult();
-            try (PreparedStatement stmt = conn.prepareStatement(queryForDDPParticipantId)) {
+            try (PreparedStatement stmt = conn.prepareStatement(queryForDDPParticipantId,
```

Comment:
@MocanaAtBroad the pattern seems to be that with `8.0.20`, we have to set cursor modes explicitly for places that are going backwards through cursors.  I did a find/replace for areas of `rs.beforeFirst()` to try to find these.

Reply:
Had no idea that one could do these tricks with result sets! (though this looks like someone really really does not want to write more SQL queries than absolutely necessary!)